reviews/models.py

```diff
@@ -1,12 +1,24 @@
-from django.db   import models
-from core.models import TimeStampModel
+import random
+
+from django.db        import models
+from django.db.models import Max
+
+from core.models      import TimeStampModel
 
 class Review(TimeStampModel):
     user    = models.ForeignKey('users.User', on_delete = models.SET_NULL, null = True)
     room    = models.ForeignKey('stays.Room', on_delete = models.CASCADE)
     rating  = models.FloatField()
     comment = models.TextField(null = True)
 
+    @classmethod
+    def get_random(cls, reviews):
+        if not reviews:
+            return None
+        num    = random.randrange(reviews.count())
+        review = reviews[num]
+        return review
```

Comment:
3줄로 수정해도 충분히 가독성 좋을 것 같습니다!
```suggestion
    def get_random(cls, reviews):
        if not reviews:
            return None
            
        return reviews[random.randrange(reviews.count())]
```